src/main/resources/com/chikli/hudson/plugin/naginator/NaginatorOptOutProperty/help-optOut.html

```diff
@@ -1,4 +1,4 @@
 <div>
-    By default, naginator offer a "rebuild" link to all failed builds. Some jobs anyway might not be designed to
+    By default, naginator offers a "retry" link to all failed builds. Some jobs anyway might not be designed to
```

Comment:
Change note: `retry` is correct: https://github.com/jenkinsci/naginator-plugin/blob/naginator-1.18.2/src/main/java/com/chikli/hudson/plugin/naginator/NaginatorRetryAction.java#L51